client/gutenberg/extensions/publicize/form-unwrapped.jsx

```diff
@@ -75,7 +74,7 @@ class PublicizeFormUnwrapped extends Component {
 	isConnectionOn( uniqueId ) {
 		const { activeConnections } = this.props;
 		const matchingConnection = activeConnections.find( c => uniqueId === c.unique_id );
-		if ( isNil( matchingConnection ) ) {
+		if ( matchingConnection == null ) {
```

Comment:
Hmm, not a big fan of using untyped `==` -- we generally discourage that in Calypso (and even have an eslint rule, don't we?)

Which values of `matchingConnection` will this be `true` for? `null` and `'null'`? What about `undefined`? Or anything falsey?

Can we replace it by a simple `if ( matchingConnection )` check? I'd be kinda surprised we'd need something as specific as `== null` here when we don't anywhere else in Calypso really...

Reply:
It didn't take a lot to convince me TBH 👍 😆 

Done in https://github.com/Automattic/wp-calypso/pull/28017/commits/b6cc8acd63cf0b18c86179dda8c491eb092db24b